articles/faq/stay-on-plan-cost-more.md

```diff
@@ -1,19 +1,25 @@
 ---
 title: "If I stay on my current plan, will my monthly cost stay the same?"
-description: "Your monthly cost will remain the same for the lifetime of your account. However, your overages will be capped at your maximum sending from the last 90 days. If you intend to send more than your limit, we’d encourage you to take advantage of the discounts, and upgrade your account."
+description: "Your monthly cost will remain the same for the lifetime of your account. However, your overages will be capped at your maximum sending from the last 90 days plus 30%. If you intend to send more than your limit, we’d encourage you to take advantage of the discounts, and upgrade your account."
 ---
 
-Your monthly cost will remain the same for the lifetime of your account. However, your overages will be capped at your maximum sending from the last 90 days or the published plan volume, whichever is the greater. If you intend to send more than your limit, we’d encourage you to take advantage of the discounts, and upgrade your account.
+Your monthly cost will remain the same for the lifetime of your account. However, your overages will be capped at your maximum sending from the last 90 days or the published plan volume plus 30%, whichever is the greater. If you intend to send more than your limit, we’d encourage you to take advantage of the discounts, and upgrade your account.
 
-For example if you are on a 1M plan and have the following sending pattern for the last 90 days:
+
+**Example 1 - No overage**
+If you are on a 1M plan and have not overaged in the last 90 days your max monthly cap will be set to 1.3M (1M x1.3)
+
+**Example 2 - Overage**
+If you are on a 1M plan and have the following sending pattern for the last 90 days:
 * Aug 19 to Sep 18: 1,136,456
```

Comment:
Can we simplify these US-centric date periods to just month names?

```
* August: 1,136,456
* July: 1,232,012
* June: 986,472
```

Reply:
I can extend them to August - but we need to include the days - as that is the whole point